website/registrations/models.py

```diff
@@ -108,7 +108,7 @@ def payment_amount(self):
     @property
     def payment_payer(self):
         if self.membership:
-            return self.membership.user
+            return PaymentUser.objects.get(pk=self.membership.user.pk)
```

Comment:
Why does this need to be a PaymentUser exactly? The place where this property is used could convert it, then you don't need to import the model here